pages/myposts.vue

```diff
@@ -15,7 +15,11 @@
             :chats="me.expectedchats"
           />
           <div v-if="!me" class="d-flex justify-content-center mt-4 flex-wrap">
-            <b-button variant="primary" size="lg" @click="forceLogin">
+            <b-button
+              variant="primary"
+              size="lg"
+              @click="authStore.forceLogin = true"
```

Comment:
The method was only used once - **inlined**

Reply:
I understand why you would do this, but I prefer separate methods please.

Reply:
Reverted.